webb_ai/README.md

```diff
@@ -0,0 +1,56 @@
+# Webb.ai
+
+## Overview
+
+Matt by Webb.ai is the first AI-enabled reliability engineer.
+Matt troubleshoots Monitors from Datadog, incidents from infrastructure like Kubernetes, 
+and alerts from cloud providers like AWS, Azure, and Google Cloud.
+It automates troubleshooting to identify the root cause of an alert or incident in less than 5 mins.
+
+With this integration, MTTR for Datadog customers will be reduced as Matt identifies Datadog alerts and automatically troubleshoots them, typically in less than 5min
+
+Data queried by Matt:
+- Events to identify Monitors in alert state
+- Monitors, Metrics and their values
+
+Matt sends the following events to Datadog:
+- RCAs performed by Matt
+- Changes observed in Kubernetes clusters
```

Comment:
This is a great start but it would be helpful to provide the value in 1-2 sentences of the data being monitored. EX: 

This integration sends events to Datadog including RCAs performed by Matt and changes in kubernetes clusters <insert value of this information here> 

Ex of value: providing insight into xyz

Reply:
We want to inform the user as to why this information is helpful to them

Reply:
@emarsha94 Thanks for the feedback. We updated the text to reflect why we think this is useful to datadog users. Please take a look.